src/HTTPResponse.cpp

```diff
@@ -44,12 +44,16 @@ void HTTPResponse::setHeader(const std::string &name, const std::string &value)
 
```

Comment:
I see reformatting here due to auto-format. Did you manage to install clang-format properly also on the school computer?

Reply:
Yeah I have clang